tests/features/menus/feature_set.feature

```diff
@@ -0,0 +1,22 @@
+@api
+Feature: Feature set menu
+  In order to easily enable a feature
```

Comment:
Provided feature description is not inline with provided scenario. Imho this is also out of the scope for the NEPT-182.